ext/mbstring/libmbfl/filters/mbfilter_htmlent.c

```diff
@@ -67,6 +67,7 @@ const mbfl_encoding mbfl_encoding_html_ent = {
 	&vtbl_wchar_html,
 	mb_htmlent_to_wchar,
 	mb_wchar_to_htmlent,
+	NULL,
 	NULL
```

Comment:
Nit: Using trailing commas would lead to nicer diffs in the future :slightly_smiling_face:

Reply:
Very good point. I can fix that in this PR.

Reply:
Done.